os/src/ProcessOps.scala

```diff
@@ -40,7 +38,38 @@ object call {
       timeout = timeout,
       check = check,
       propagateEnv = propagateEnv,
-      timeoutGracePeriod = timeoutGracePeriod
+      shutdownGracePeriod = shutdownGracePeriod,
+      destroyOnExit = destroyOnExit
+    )
+  }
+  def apply(
```

Comment:
Looks like this one is only for bin-compat, so we should deprecate it for removal.